README.md

```diff
@@ -55,23 +55,24 @@ To run the tests using `pytest`, you need to have the cloned mne-python reposito
 with a local pip-install instead of the mne-python package from pypi. Update your
 installation as follows:
 
-    $ git clone https://github.com/mne-tools/mne-python
+    $ git clone https://github.com/mne-tools/mne-python --depth 1
```

Comment:
why this?

Reply:
why should we download the complete git history if we are only interested in making test data available?

Reply:
maybe you want to make a PR to mne-python? :) Is mne-python clone that heavy?

Reply:
it has all the test data in the history before some of it was moved elsewhere

Reply:
I just checked ... cloning with depth 1 gives a `--du` of 149M ... cloning the complete history gives 196M

Reply:
worth it or not? give it a thumbs up or thumbs down ...

Reply:
oh wow, I thought it used to be ~80 MB download with data. But maybe it's changed. Okay to recommend `--depth 1`

Reply:
you can do much simpler than this:

`pip install -U https://api.github.com/repos/mne-tools/mne-bids/zipball/master`

no need to have git on your machine and you get the latest version.

Reply:
does this also provide the data in `mne-python/io/tests`?

Reply:
try. I am not sure.

Reply:
Does not work unfortunately, I'll go with git and depth 1

`FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/miniconda/envs/testenv/lib/python3.6/site-packages/mne/io/kit/tests/data/test.sqd'`